docs/governance/governance_model_v1_archive.rst

```diff
@@ -0,0 +1,263 @@
+Governance model v1 - archive
+#############################
+
+.. vale off
+
+Following the :xref:`Mautic acquisition announcement` in May 2019, which included all assets relating to the Mautic community, a governance model was proposed and adopted following a period of community consultation.
+
+Read more about it in the :xref:`Mautic Community Governance Model initial proposal` and the :xref:`Governance Model Proposal subsequent follow up` which addressed questions raised by the community. Subsequent adaptations were made during the Community Summit in November 2019 to arrive at the model outlined below.
+
+Further updates were made in May 2020 with the change in project leadership and introduction of the Founder role.
+
+The aim of the governance model is to enable and empower volunteers. Mautic wants to encourage the community to take up responsibilities and have a say in the direction of the Mautic Open Source project, while working collaboratively with Acquia.
+
+Community structure
+*******************
+
+The structure of the community includes five teams, each led by a Team Lead and Assistant Team Lead, with working groups under them.
+
+* :doc:`/teams/community_team/community_team`
+* :doc:`/teams/education_team/education_team`
+* :doc:`/teams/legal_and_finance_team/legal_and_finance_team`
+* Marketing Team
```

Comment:
Commenting for reminder that this should be added once the "Marketing Section" has been converted.

Reply:
Let's make sure it's captured as an issue so we don't lose sight of this.

Reply:
It's in issue #330.